### DISCUSSION TITLE
Add flame chart panel type

### DISCUSSION THREAD

**Discussion by tomershafir** [grafana/grafana#71752] [Dashboards, Panels & Visualizations]
Flame chart, where x variable represents time, may be useful to observe profile patterns across time.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). To facilitate this, I will be closing this discussion.

Please check out this documentation for the flame graph https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/flame-graph/

If this doesn't suit your needs, feel free to make an [issue](https://github.com/grafana/grafana/issues) to request the feature you would like. Thank you!
